pkgs/tools/text/rst2html5/default.nix

```diff
@@ -9,6 +9,7 @@ in python3Packages.buildPythonPackage {
 
   src = python3Packages.fetchPypi {
     inherit pname version;
+    format = "wheel";
```

Comment:
or `inherit format` ?

Reply:
Don't know how I managed to miss that, amended.